pkg/server/grpc/tracee.go

```diff
@@ -138,10 +706,19 @@ func convertDefinitionToProto(d events.Definition) *pb.EventDefinition {
 	}
 }
 
+func getGlobalID(e trace.Event) pb.EventId {
+	idExternal := EventTranslationMap[events.ID(e.EventID)]
+	if idExternal != 0 {
+		return pb.EventId(idExternal)
+	}
+	return pb.EventId_unspecified
```

Comment:
In case the mapping doesn't exist the event might be a non "built-in" event of trace (higher than 10,000), so I think we should return the event id as is (enums in proto 3 are considered "open" enums, so it should be possible to specify id's out of the enum range)

Reply:
Agreed, so I think here we have 2 possible cases:
1. if below 10,000, check the translation table in order to get the external ID. If no external ID is found, we should return `pb.EventId_unspecified`.
2. If is higher than 10,000, we don't need to check the translation table and simple return id as is.

Have I doubt for 1: if event ID is not defined in the translation table, we always return `pb.EventId_unspecified` instead of id as id, right?